pyproject.toml

```diff
@@ -101,21 +101,21 @@ build-frontend = "build"
 build-verbosity = 1
 test-extras = ["dev"]
 test-command = "pytest {package}/tests"
+# we do not support PyPy, musllinux
+skip = ["pp*", "*musllinux*"]
```

Comment:
why not?

Reply:
PyPy is very tricky to get it working. I tried once but there are so many errors so I decided not to prioritize it.

`musl` is not supported by `numpy` yet. It does not make sense that we support it. Numpy 1.25 will support `musl`, we can then also add the support.
https://github.com/numpy/numpy/releases/tag/v1.25.0rc1